### PR TITLE
chror : Fix types popconfirm , page-header ...

### PR DESCRIPTION
**Pull request by cydiacen**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: types/page-header.d.ts [hunk 1]
```diff
@@ -3,59 +3,60 @@
...
+import { VNodeChild } from 'vue';
 
 export declare class PageHeader extends AntdComponent {
   /**
    * Custom backIcon
    * @default <ArrowLeftOutlined />
    * @type any (string | slot)
    */
-  backIcon: any;
+  backIcon?: string | VNodeChild | JSX.Element;
```
Comment (Amour1688) [inline types/page-header.d.ts:14]: `VNodeChild` 包含了 `string`

File: types/page-header.d.ts [hunk 2]
```diff
@@ -3,59 +3,63 @@
...
+    extra?: VNodeChild | JSX.Element;
+
+    avatar?: object;
+
+    ghost?: boolean;
+
+    /**
+     * Specify a callback that will be called when a user clicks backIcon.
+     */
+    back(): void;
```
Comment (tangjinzhou) [inline types/page-header.d.ts:62]: 这个是事件

File: types/popconfirm.d.ts [hunk 3]
```diff
@@ -3,41 +3,41 @@
 // Definitions: https://github.com/vueComponent/ant-design-vue/types
 
 import { TooltipCommon } from './tootip/common';
-
+import { VNodeChild } from 'vue';
 export declare class Popconfirm extends TooltipCommon {
   /**
```
Comment (tangjinzhou) [inline types/popconfirm.d.ts:8]: $props

File: types/radio/radio-button.d.ts [hunk 4]
```diff
@@ -9,5 +9,5 @@ export declare class RadioButton extends Radio {
    * Type of radio button
    * @type string
    */
-  type: 'primary' | 'danger' | 'dashed' | 'ghost' | 'default';
+  type?: 'primary' | 'danger' | 'dashed' | 'ghost' | 'default';
```
Comment (tangjinzhou) [inline types/radio/radio-button.d.ts:12]: 这种extend 的  $props 怎么处理

File: types/radio/radio-group.d.ts [hunk 5]
```diff
@@ -5,48 +5,50 @@
...
-   */
-  value: any;
-
-  /**
-   * style type of radio button
-   * @default 'outline'
-   * @type string
-   */
-  buttonStyle: 'outline' | 'solid';
+    defaultValue?: boolean;
```
Comment (tangjinzhou) [inline types/radio/radio-group.d.ts:13]: any

File: types/radio/radio-group.d.ts [hunk 6]
```diff
@@ -5,48 +5,50 @@
...
+     */
+    value?: any;
+
+    /**
+     * style type of radio button
+     * @default 'outline'
+     * @type string
+     */
+    buttonStyle?: 'outline' | 'solid';
+  }
```
Comment (tangjinzhou) [inline types/radio/radio-group.d.ts:53]: 添加 onChange  事件

File: types/select/select.d.ts [hunk 7]
```diff
@@ -5,248 +5,249 @@
...
+     * @default true
+     * @type boolean
+     */
+    dropdownMatchSelectWidth?: boolean;
+
+    /**
+     * Customize dropdown content
+     * @type function | slot-scope
+     */
+    dropdownRender?: (menu?: VNodeChild, props?: object) => VNodeChild;
```
Comment (tangjinzhou) [inline types/select/select.d.ts:72]: 改成单参数吧

File: types/select/select.d.ts [hunk 8]
```diff
@@ -5,248 +5,249 @@
...
+
+    /**
+     * style of dropdown menu
+     * @type object
+     */
+    dropdownStyle?: object;
```
Comment (tangjinzhou) [inline types/select/select.d.ts:78]: CSSProperties

File: types/statistic/statistic-countdown.d.ts [hunk 9]
```diff
@@ -3,32 +3,34 @@
...
-   * @type string or number
-   */
-  value: string | number;
-  valueStyle: object;
+    /**
+     * Display value
+     * @type string or number
+     */
+    value?: string | number;
+    valueStyle?: object;
```
Comment (tangjinzhou) [inline types/statistic/statistic-countdown.d.ts:34]: CSSProperties

File: types/pagination.d.ts [hunk 10]
```diff
@@ -6,89 +6,91 @@ import { AntdComponent } from './component';
...
-  /**
-   * to customize item innerHTML
-   * @type Function
-   */
-  itemRender?: (page: number, type: 'page' | 'prev' | 'next', originalElement: any) => VNode;
+    /**
+     * to customize item innerHTML
+     * @type Function
+     */
+    itemRender?: (page: number, type: 'page' | 'prev' | 'next', originalElement: any) => VNode;
```
Comment (tangjinzhou) [inline types/pagination.d.ts:94]: 改成单参数 {page, type, originalElement}

**Pull request merged**